src/Checkbox/checkbox.style.ts

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { Colors } from "..";
+import mapping from "../../public/fonts/vivy-icons/iconMapping.json";
+
+const checkboxEmpty = mapping["icon-checkbox-empty"];
+const checkboxFilled = mapping["icon-checkbox-filled"];
+
+export const CheckBoxWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const CheckMark = styled.label`
+  padding: 0;
+  font-size: 24px;
+  font-weight: normal;
+  line-height: 1.38;
+  letter-spacing: normal;
+  color: #c9c9c9;
+  margin-right: 16px;
+  cursor: pointer;
+`;
+
+export const CheckBoxInput = styled.input.attrs({ type: "checkbox" })`
+  display: none;
+  + ${CheckMark}:after{
+    font-family: "vivy-icons";
+    content: "\\${checkboxEmpty}";
+  }
+  &:checked + ${CheckMark}:after {
+    font-family: "vivy-icons";
+    content: "\\${checkboxFilled}";
+  }
+`;
+
+export const Label = styled.label`
+  font-size: 16px;
+  line-height: 1.38;
+  letter-spacing: normal;
```

Comment:
hmm, do we need this `letter-spacing` ad I didnt see any changes  without it? 
<img width="1092" alt="Screenshot 2020-07-08 at 17 39 18" src="https://user-images.githubusercontent.com/26152898/86939566-08ed6780-c142-11ea-9772-c74b15bb00bc.png">

Reply:
This screenshot is not really useful since I don't have any point of reference. It might be that the letter spacing definition here is superfluous but then again, is this checkbox component used somewhere where we need this? Since this is a migration and somebody put this in on purpose, I don't see any harm in leaving this in.

Reply:
hmm, yeah I had to make a gif actually but it's ok, let's leave it as it is if that ok, but the issue, cause I thought it's only affecting the label of the checkbox and that's why I found it useless